kubeflow/fairing/ml_tasks/tasks.py

```diff
@@ -67,29 +67,56 @@ def _build(self):
 
 
 class TrainJob(BaseTask):
-
+    """
+    Create a train job.
+    """
     def __init__(self, entry_point, base_docker_image=None, docker_registry=None,  # pylint:disable=useless-super-delegation
                  input_files=None, backend=None, pod_spec_mutators=None):
+        """
+        Init the Train Job class.
+        args:
```

Comment:
@QxiaoQ Could you please update here to use uniform format? Thanks a lot!

Reply:
Format use pyment. Thanks.